### DISCUSSION TITLE
How to do allowUnfree in flake template?

### DISCUSSION THREAD

**Discussion by gbirke** [nix-community/nixvim#1062] [Q&A] [answered]
I'd like to use the `intelephense` LSP, which has an unfree license. I'm quite new to Nix and cannot figure out where in the flake for NixVim (generated with the [template](https://github.com/nix-community/nixvim/blob/main/templates/simple/flake.nix)) I can put `config.allowUnfree = true;`

**Comment by traxys** [comment, marked as answer]
I think you need someting like this:
https://github.com/hercules-ci/flake-parts/blob/main/template/unfree/flake.nix#L13

I'm not too sure though, I haven't tested

**Reply by gbirke** [reply]
Added that snippet to my `flake.nix`, could build without a hitch.
